tests/test_utils.py

```diff
@@ -5,10 +5,10 @@
 from typing import Iterator, Tuple
 from typing import Type as TypingType
 
+import matplotlib.pyplot as plt
```

Comment:
y?

Reply:
just for consistency, i noticed we were doing it both ways